COPYRIGHT

```diff
@@ -17,3 +17,10 @@ of their contributions is retained by an organisation:
       King's College London
     Laurence Tratt <laurie@tratt.net>: copyright retained by
       King's College London
+
+
+run_dacapo.py is a derived work of a UPL licensed file:
+https://github.com/softdevteam/warmup_experiment/blob/a7518de845f4a540015caa2722b81183c0e52483/extbench/rundacapo.py
```

Comment:
It would be better for us to relicense warmup_experiment than to have to do this particular dance?

Reply:
Can that be done after the fact? If so, yes, let's.

Reply:
I'll get to it shortly.

Reply:
I forgot to post the PR link https://github.com/softdevteam/warmup_experiment/pull/473